### DISCUSSION TITLE
XCT Popup How to disable dimming on popup?

### DISCUSSION THREAD

**Discussion by Eden-Mor** [xamarin/XamarinCommunityToolkit#1692] [Q&A]
I wanted to know if it was possible to disable the dimming
![image](https://user-images.githubusercontent.com/20691183/139553753-7f8ba6bd-7c2a-48fd-b3f2-f5e0a912d230.png)
I want the background to be transparent.
I tried setting the background and background color to transparent but that doesnt do anything.
I messed with shadows and other options I could find to no avail.
![image](https://user-images.githubusercontent.com/20691183/139553789-441b151a-669b-4446-9dd2-79d9e9aeb91e.png)
The above is what im trying to achieve.
Thank you.

**Comment by pictos** [comment]
@ahoefling do you know if it's possible?

**Reply by SkyeHoefling** [reply]
Take a look at the sample code for Transparent Popups, you should be able to set the `Color` property on the Popup Control.

https://github.com/xamarin/XamarinCommunityToolkit/blob/main/samples/XCT.Sample/Pages/Views/Popups/TransparentPopup.xaml

If that doesn't get the desired effect you can create a custom renderer that adjust the Android Dim Background

**Reply by Eden-Mor** [reply]
Thank you for the reply, Unfortunately Color=Transparent does not work (atleast not for android)
I will try to setup that custom renderer, this would be my first time so this should be fun!

**Reply by SkyeHoefling** [reply]
I just took a look at the original PR #653 to see the screenshots of what works and what doesn't work when I first submitted it. Out of the box Android still uses the default dim background even in the transparent popup sample. The easiest way to solve this is create a custom popup renderer that sets the dim background on android to transparent.

If you have the time, would love to see a PR that uses an attached behavior so others can get a similar effect.
